webhook/resourcesemantics/conversion/conversion.go

```diff
@@ -109,11 +109,11 @@ func (r *reconciler) convert(
 	out := outZygote.DeepCopyObject().(ConvertibleObject)
 
 	hubGVK := inGVK.GroupKind().WithVersion(conv.HubVersion)
-	logger = logger.With(
+	ctx = logging.WithLogger(ctx, logger.With(
```

Comment:
Given this logger was not used at all, may be we just delete it?

Reply:
@dprotaso will know which way is best I'd hope.

Reply:
Thanks, I had vaguely recalled there being one of these.  In general, I think this is probably the right change.